### PR TITLE
[SE-3844] add crafty-bot role

### PR DESCRIPTION
**Pull request by gabor-boros**
This PR adds a new role for the crafty bot.

**Dependencies**: None

**Merge deadline**: None

**Testing instructions**:

N/A

**Author notes and concerns**:

N/A

**Reviewers**
- [x] @Agrendalath

### CODE REVIEW

File: playbooks/roles/crafty-bot/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    mode: 0600
+    backup: yes
+  become_user: ubuntu
+
+- name: Start Crafty Bot with docker-compose
+  docker_service:
+    project_src: "{{ APP_PATH }}"
+    pull: yes
+    debug: yes
+  become_user: ubuntu
```
Comment (Agrendalath) [inline playbooks/roles/crafty-bot/tasks/main.yml:67]: That's a lot of duplicated code. Should we consider moving it to a common playbook, e.g. something like `docker-compose-deploy`? We can add it as a TODO.

File: playbooks/roles/crafty-bot/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+
+# Settings for Crafty Bot. See crafty_bot/config/settings/ for details.
+ENV_TOKENS:
+  # General
+  # ------------------------------------------------------------------------------
+  # DJANGO_READ_DOT_ENV_FILE: True
+  DJANGO_SETTINGS_MODULE: config.settings.production
+  DJANGO_SECRET_KEY: set-me-please
+  DJANGO_ADMIN_URL: admin/
+  DJANGO_ALLOWED_HOSTS: .crafty.opencraft.com
```
Comment (Agrendalath) [inline playbooks/roles/crafty-bot/defaults/main.yml:22]: Nit: this is already specified in the private vars, so we can make the value more generic here.

**Review by Agrendalath — APPROVED** [review]
Please address my comments before merging.

:+1:
* [x]  I tested this: read through the playbooks
* [x]  I read through the code
* [x]  I checked for accessibility issues: n/a
* [x]  Includes documentation: n/a

File: playbooks/roles/crafty-bot/tasks/main.yml (hunk 1 above)
Comment (gabor-boros) [reply playbooks/roles/crafty-bot/tasks/main.yml:67]: Created SE-3919 to mitigate this.

**Pull request merged**